### PR TITLE
Removes the Construction Worker job, moves tools into QM Crate

### PR DESCRIPTION
**Pull request by comradef191**
[REMOVAL]

## About the PR 
Removes the Construction Worker-not Station Builder-role completely. 
Adds a new Engineering Locked QM crate for 8,000c, containing: Floor and Wall Planner, Window Planner, Lamp Manufacturer and Conworker Overalls.

## Why's this needed? 
The Construction Worker is heavily overpowered and abused, this is due to the fact they start fully spacesuited, have a full set of tools _and_ have an RCDD. The role is very much not intended for general play, and has no real reason to remain. 

## Changelog

```changelog
(u)comradef191:
(*)Removed the Construction Worker job. Added QM crate containing most of their equipment.
```

### CODE REVIEW

**Comment by MarkNais** [comment]
Can we get some examples of the abuse please?

**Comment by Carbadox** [comment]
Seems like a reactionary change honestly. For the time I've played I haven't had a directly greifing construction worker ever. Sure, id see them doing annoying stuff like making mazes in the corridors but anyone can do that with just steel sheets. 

I don't want to sound cliche by saying "improve and don't remove", but I think that Construction Worker kinda deserves a lot better than just being axed.

I have an idea for a PR that I want to do regarding him, no promises for when but I just wanted to say that so that it doesn't come off as me saying "don't remove" without suggesting a different alternative

**Comment by TTerc** [comment]
Have you tried adminhelping people who abuse this job in any way?

Removing the whole job just because you've seen people grief as conworkers smells like grudgecode imo, the only thing that would need to be tweaked is the floor and wall planner, griefy textures should be gated behind emag

Personally I haven't witnessed any cases of conworker grief really (aside from making the station ugly) and I definetly have seen people being really creative with their building projects so I'm strongly against it

File: code/datums/supply_packs.dm [hunk 1]
```diff
@@ -1179,6 +1179,17 @@
 	containertype = /obj/storage/crate/classcrate/qm
 	containername = "Anti-Singularity Supply Pack"
 
+/datum/supply_packs/conworksupplies
+	name = "Construction Equipment"
+	desc = "The mothballed tools of our former Construction Workers, in a crate, for you!"
+	category = "Engineering Department"
+	contains = list(/obj/item/lamp_manufacturer/organic,/obj/item/material_shaper,/obj/item/room_planner,/obj/item/clothing/under/rank/orangeoveralls)
+	//i was going to add a version of the construction visualliser w/o seeing invisible monsters but FUCK SIGHT CODE WHAT THE FUCK
```
Comment (Tarmunora) [inline code/datums/supply_packs.dm:1187]: 😔

**Comment by nefarious6th** [comment]
I left some thoughts on Uranium's PR that last worked on the CWorker stuff. Honestly? I think a timegate is a fair solution, since Ghostdrones are also gated but serve in a relatively similar function with regards to building capability/supplies.
I think we see some grief and bad from CWorkers; but I have also lost count of how many times a CWorker, attentive, fast, and equipped as ever, has been able to respond to massive breaches and patch them within minutes. Removing CWorker outright will, certainly, leave some unfillable gaps, in my opinion. CWorkers are also super useful last-ditch efforts for fixing subverted AIs when someone _really_ worked to make it near-impossible to reset.

File: code/datums/supply_packs.dm (hunk 1 above)
Comment (Sovexe) [reply code/datums/supply_packs.dm:1187]: for real though sight code is a disaster

**Comment by Tarmunora** [comment]
One way or another, the RCD-Deluxe *should* be removed from being an item that you can get, at least easily. The regular RCDs are head-level equipment.

**Comment by Azrun** [comment]
Building off of nefarious6th's post. I like late_join + [structural_damage calculation](https://github.com/goonstation/goonstation/blob/f554c069a216e8dff200903f0b5ea6f62a5b9a79/code/procs/scorestats.dm#L102-L125) and/or have it something _Heads_ can [request from COMMaster](https://github.com/goonstation/goonstation/blob/master/code/modules/networks/computer3/communication.dm).

**Comment by comradef191** [comment]
OOops

**Review by Tarmunora — APPROVED** [review]

**Comment by Tarmunora** [comment]
Construction worker may or may not come back in the future in some form, but for now, it's being rotated out of job selection.

**Comment by comradef191** [comment]
ATLAST
AFTER 4 AND A HALF MONTHS
MERGED

**Pull request merged**